wasm-binary.cpp:1460:12: error: comparison is always false due to limited range of data type (aarch64, armhf, ppc64le)

I’ve updated binaryen package from 1.37.12 to 1.37.14 and now it fails to build on aarch64, armhf and ppc64le:

```
Building CXX object src/passes/CMakeFiles/passes.dir/Untee.cpp.o
cd /home/buildozer/aports/community/binaryen/src/binaryen-1.37.14/src/passes && /usr/bin/c++   -I/home/buildozer/aports/community/binaryen/src/binaryen-1.37.14/src  -Os -std=c++11 -Wall -Werror -Wextra -Wno-unused-parameter -fno-omit-frame-pointer -fPIC -O3 -DNDEBUG -O2 -UNDEBUG   -o CMakeFiles/passes.dir/Untee.cpp.o -c /home/buildozer/aports/community/binaryen/src/binaryen-1.37.14/src/passes/Untee.cpp
/home/buildozer/aports/community/binaryen/src/binaryen-1.37.14/src/wasm/wasm-binary.cpp: In function 'int32_t wasm::readBase64VLQ(std::istream&)':
/home/buildozer/aports/community/binaryen/src/binaryen-1.37.14/src/wasm/wasm-binary.cpp:1460:12: error: comparison is always false due to limited range of data type [-Werror=type-limits]
     if (ch == EOF)
            ^
```

--- a/src/wasm/wasm-binary.cpp
+++ b/src/wasm/wasm-binary.cpp
@@ -1456,7 +1456,7 @@ static int32_t readBase64VLQ(std::istream& in) {
   uint32_t value = 0;
   uint32_t shift = 0;
   while (1) {
-    char ch = in.get();
+    auto ch = in.get();
     if (ch == EOF)
       throw MapParseException("unexpected EOF in the middle of VLQ");
     if ((ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch < 'g')) {

